Frontend/src/Components/Faults/FaultsView.js

```diff
@@ -285,185 +285,384 @@ export default function Faults(props) {
+          <Box h={imageHeight} />
+        )}
+        {props.data?.mc_status[0] > CONSTANTS.mc_status.MIN ? (
+          <Tooltip label={getMotorControllerString()}>
+            <Image fit={fitType} boxSize={imageHeight} src={Images.MCStatus} />
+          </Tooltip>
+        ) : (
+          <Box h={imageHeight} />
+        )}
+        {props.data?.pack_current[0] > CONSTANTS.pack_current.MAX ? (
+          <Tooltip label={"High battery pack current"}>
+            <Image fit={fitType} boxSize={imageHeight} src={Images.HighCurrent} />
+          </Tooltip>
+        ) : props.data?.pack_current[0] < CONSTANTS.pack_current.MIN ? (
+          <Tooltip label={"Low battery pack current"}>
+            <Image fit={fitType} boxSize={imageHeight} src={Images.LowCurrent} />
+          </Tooltip>
+        ) : (
+          <Box h={imageHeight} />
+        )}
+        {props.data?.pack_voltage[0] > CONSTANTS.pack_voltage.MAX ? (
+          <Tooltip label={"High battery pack voltage"}>
+            <Image fit={fitType} boxSize={imageHeight} src={Images.HighVoltage} />
+          </Tooltip>
+        ) : props.data?.pack_voltage[0] < CONSTANTS.pack_voltage.MIN ? (
+          <Tooltip label={"Low battery pack voltage"}>
+            <Image fit={fitType} boxSize={imageHeight} src={Images.LowVoltage} />
+          </Tooltip>
+        ) : (
+          <Box h={imageHeight} />
+        )}
+        {props.data?.mppt_current_out[0] > CONSTANTS.mppt_current_out.MAX ? (
+          <Tooltip label={"High MPPT current"}>
+            <Image fit={fitType} boxSize={imageHeight} src={Images.MPPTCurrent} />
+          </Tooltip>
+        ) : props.data?.mppt_current_out[0] < CONSTANTS.mppt_current_out.MIN ? (
+          <Tooltip label={"MPPT current is negative"}>
+            <Image fit={fitType} boxSize={imageHeight} src={Images.MPPTCurrent} />
+          </Tooltip>
+        ) : (
+          <Box h={imageHeight} />
+        )}
+        {props.data?.bms_input_voltage[0] > CONSTANTS.bms_input_voltage.MAX ? (
+          <Tooltip label={"High BMS input voltage"}>
+            <Image
+              fit={fitType}
+              boxSize={imageHeight}
+              src={Images.BMSInputVoltage}
+            />
+          </Tooltip>
+        ) : props.data?.bms_input_voltage[0] < CONSTANTS.bms_input_voltage.MIN ? (
+          <Tooltip label={"Low BMS input voltage"}>
+            <Image
+              fit={fitType}
+              boxSize={imageHeight}
+              src={Images.BMSInputVoltage}
+            />
+          </Tooltip>
+        ) : (
+          <Box h={imageHeight} />
+        )}
+        {props.data?.soc[0] < CONSTANTS.soc.MIN ? (
+          <Tooltip label={`Battery charge is low (<${CONSTANTS.soc.MIN}%)`}>
+            <Image fit={fitType} boxSize={imageHeight} src={Images.LowBattery} />
+          </Tooltip>
+        ) : (
+          <Box h={imageHeight} />
+        )}
+        {checkCellGroupVoltages() ? (
+          <Tooltip label={getCellGroupVoltageString()}>
```

Comment:
Consider adding the same props as the temperature tooltip (`shouldWrapChildren` and `size`)

Reply:
Changed